tests/common-tests.h

```diff
@@ -3640,16 +3640,45 @@ TEST_CASE_METHOD(common_tests, "Basic logging support", "[core][logging]")
     catch (...) {}
 
     CHECK(sql.get_last_query() == "drop table soci_test1");
+    CHECK(sql.get_last_query() == sql.get_last_query_with_context());
 
     sql.set_log_stream(NULL);
 
-    try
     {
-        sql << "drop table soci_test2";
-    }
-    catch (...) {}
+        auto_table_creator tableCreator(tc_.table_creator_1(sql));
 
-    CHECK(sql.get_last_query() == "drop table soci_test2");
+        int id = 1;
+        std::string name = "b";
+        sql << "insert into soci_test (name,id) values (:name,:id)", use(name, "name"), use(id, "id");
+
+        CHECK(sql.get_last_query() == "insert into soci_test (name,id) values (:name,:id)");
+        CHECK(sql.get_last_query_with_context() == "insert into soci_test (name,id) values (:name,:id) with :name=\"b\", :id=1");
```

Comment:
Pretty minor too, but I think using raw string literals is more readable (and allows copying and pasting easily, if necessary) than escaping quotes, i.e.
```suggestion
        CHECK(sql.get_last_query_with_context() == R"(insert into soci_test (name,id) values (:name,:id) with :name="b", :id=1)");
```